From f4ede888bb153b19647780fe09218fad888007ad Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?D=2E=20Andrei=20M=C4=83ce=C8=99?= <dmaces@nd.edu>
Date: Sun, 17 Dec 2017 17:47:41 +0200
Subject: [PATCH 2/3] libexternal: Add libbase shared library dependency

This should also make the required android-base/unique_fd.h ready. Inspired by: https://android-review.googlesource.com/#/c/platform/hardware/qcom/display/+/216595/

Change-Id: I473ce7d0b9b2c1182d839c3faa87819ef93834c1
---
 msm8974/libexternal/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/msm8974/libexternal/Android.mk b/msm8974/libexternal/Android.mk
index a82037a..b5c1c0e 100644
--- a/msm8974/libexternal/Android.mk
+++ b/msm8974/libexternal/Android.mk
@@ -7,6 +7,7 @@ LOCAL_MODULE_PATH             := $(TARGET_OUT_SHARED_LIBRARIES)
 LOCAL_MODULE_TAGS             := optional
 LOCAL_C_INCLUDES              := $(common_includes) $(kernel_includes)
 LOCAL_SHARED_LIBRARIES        := $(common_libs)
+LOCAL_SHARED_LIBRARIES        += libbase
 LOCAL_CFLAGS                  := $(common_flags) -DLOG_TAG=\"qdexternal\"
 LOCAL_ADDITIONAL_DEPENDENCIES := $(common_deps)
 LOCAL_SRC_FILES               := external.cpp
-- 
2.7.4

